packages/scandipwa/src/component/Field/Field.component.js

```diff
@@ -193,21 +193,21 @@ export class Field extends PureComponent {
         const {
             type,
             setRef,
-            attr: { defaultChecked = false, ...newAttr } = {},
+            attr: { ...newAttr } = {},
             events: { onChange },
             events,
             isDisabled,
             label
         } = this.props;
-        const { id = '', checked, value = '' } = newAttr;
+        const { id = '', value = '' } = newAttr;
         const elem = type.charAt(0).toUpperCase() + type.slice(1);
         const inputEvents = {
             ...events,
             onChange: onChange || noopFn
         };
         // if button value is "none" do not disable
         const isButtonDisabled = (!value.match('none') && isDisabled);
-        const isChecked = isButtonDisabled || defaultChecked ? !isDisabled : null;
+        // const isChecked = isButtonDisabled || defaultChecked ? !isDisabled : null;
```

Comment:
Please don't leave the commented code.

Reply:
that comments were there already . they have now been removed